source/android/adaptivecards/src/main/res/values/attrs.xml

```diff
@@ -7,4 +7,9 @@
     <attr name="adaptiveShowCardAction" format="reference" />
     <style name="adaptiveInlineAction" format="reference" />
     <style name="adaptiveInlineActionImage" format="reference" />
+
+    <attr name="usingCustomInputs" format="boolean" />
```

Comment:
We should try to avoid requiring this flag if at all possible, not a great host developer experience.

Reply:
yes, I agree requiring that flag may not be the best dev experience but there's no way to know if an external developer has set an application wide style or drawable, this issue becomes even more evident with the spinners which have to deal with greater complications